test_lib/compaction.py

```diff
@@ -85,6 +85,15 @@ def get_compaction_strategy(node, keyspace, table):
     return compaction
 
 
+def get_table_compaction_info(keyspace: str, table: str, session: object):
+
+    query = f"SELECT compaction FROM system_schema.tables WHERE keyspace_name = '{keyspace}' AND table_name = '{table}';"
+    result = session.execute(query).one()
+
+    compaction_strategy = result.compaction if result else {}
```

Comment:
would be nice to debug log what resulted from this query - so in case of problems we could find out what was there